addons/account/models/account_payment.py

```diff
@@ -318,6 +318,10 @@ def cancel(self):
             rec.state = 'cancel'
 
     @api.multi
+    def action_set_to_draft(self):
+        self.state = 'draft'
```

Comment:
Why you need to add this method. There is already a method **action_draft** that do the same thing.